dev-client/src/screens/SitesScreen/SitesScreen.tsx

```diff
@@ -130,6 +131,15 @@ export const SitesScreen = memo(() => {
   const moveToUser = useCallback(() => {
     if (currentUserCoords !== null) {
       mapRef.current?.moveToPoint(currentUserCoords);
+      setCalloutState(
```

Comment:
Suggestion: Rename `moveToUser` to `moveToUserAndShowCallout` or something
(I had at first thought this was being re-used for moving to the user's location upon starting the app too)